tests/everest/conftest.py

```diff
@@ -144,3 +147,57 @@ def create_evaluator_server_config(run_model):
         )
 
     return create_evaluator_server_config
+
+
+@pytest.fixture
+def cached_example(pytestconfig, evaluator_server_config_generator):
+    cache = pytestconfig.cache
+
+    def run_config(
+        test_data_case: str, chdir: bool | None = False, copy: bool | None = False
```

Comment:
I am not sure the logic around `chdir` and `copy` is needed. It might be better just always to `copy` and `chdir`. At least all the uses or `cached_example` you have added have `chdir=True`.